### PR TITLE
Edits Polkadot-JS Pt2

### PR DESCRIPTION
**Pull request by filippoweb3**
Fully remove Polkadot JS bits from the Learn/Basic/Accounts section

A further PR will take care of the remaining sections in Learn/Basic

### CODE REVIEW

**Comment by filippoweb3** [comment]
@DrW3RK note that there are still missing banners in each Account page with links to dedicated JS guides and Apps (example Multisig apps for multisig accounts page).

File: docs/learn/learn-accounts.md [hunk 1]
```diff
@@ -7,67 +7,58 @@ keywords: [account, polkadot account, polkadotjs, indices, identity, reaping]
...
-[indices](./learn-account-advanced.md#indices). For a more in-depth explanation of the cryptography
-behind {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} accounts, please see
-[the cryptography page](learn-cryptography.md).
+import RPC from "./../../components/RPC-Connection"; import Tabs from "@theme/Tabs"; import TabItem
+from "@theme/TabItem"; import DocCardList from '@theme/DocCardList';
 
 :::info User friendly wallets
 
-Create your Polkadot accounts with a secure and user-friendly wallets listed on the
+Create your Polkadot accounts with a secure and user-friendly wallet listed on the
```
Comment (profradha) [inline docs/learn/learn-accounts.md]:
```suggestion
Create your Polkadot accounts with any of the secure and user-friendly wallet listed on the
```

File: docs/learn/learn-accounts.md [hunk 2]
```diff
@@ -7,67 +7,58 @@ keywords: [account, polkadot account, polkadotjs, indices, identity, reaping]
...
 [Polkadot website](https://www.polkadot.network/ecosystem/wallets/).
 
+See the [Wallets](./wallets-index) section for more information about wallets and the
+[wallets and extensions](../general/wallets-and-extensions.md) page for wallets and browser
+extensions funded by the Polkadot/Kusama Treasuries or by the
+[Web3 Foundation Grants Program](../general/grants.md).
```
Comment (profradha) [inline docs/learn/learn-accounts.md]:
```suggestion
See the [Wallets](./wallets-index) section for more information about different wallet options available, and specifically the
[wallets and extensions](../general/wallets-and-extensions.md) page, which lists the user friendly wallet projects funded by the Polkadot/Kusama Treasuries or by the
[Web3 Foundation Grants Program](../general/grants.md).
```

File: docs/learn/learn-accounts.md [hunk 3]
```diff
@@ -7,67 +7,58 @@ keywords: [account, polkadot account, polkadotjs, indices, identity, reaping]
...
-
-[Subkey](https://docs.substrate.io/reference/command-line-tools/subkey/) and Polkadot-JS based
-wallets use the BIP39 dictionary for mnemonic generation, but use the entropy byte array to generate
-the private key, while full BIP39 wallets (like Ledger) use 2048 rounds of PBKDF2 on the mnemonic.
-The same mnemonic may generate different private keys on other wallets due to the various
-cryptographic algorithms used. See
-[Substrate BIP39 Repo](https://github.com/paritytech/substrate-bip39) for more information.
-
-:::
+that do not have private keys (i.e. keyless accounts). Such accounts are
```
Comment (profradha) [inline docs/learn/learn-accounts.md]:
```suggestion
that do not have known private keys (i.e. keyless accounts). Such accounts are
```

File: docs/learn/learn-accounts.md [hunk 4]
```diff
@@ -7,67 +7,58 @@ keywords: [account, polkadot account, polkadotjs, indices, identity, reaping]
...
+[system accounts](./learn-account-advanced.md#system-accounts) that are not discussed here and are
+meant for an advanced audience.
 
-A typical mnemonic phrase generated by
-[the Subkey tool](https://docs.substrate.io/reference/command-line-tools/subkey/) is shown below.
+A typical 12-word mnemonic seed phrase is shown below.
 
 ```
-'caution juice atom organ advance problem want pledge someone senior holiday very'
+'caution juice atom organ advance problem want to pledge someone senior holiday very'
```
Comment (profradha) [inline docs/learn/learn-accounts.md]: Was this a grammarly suggestion?

```suggestion
'caution juice atom organ advance problem want pledge someone senior holiday very'
```

File: docs/learn/learn-accounts.md [hunk 5]
```diff
@@ -77,122 +68,54 @@ Secret seed (Private key): 0x056a6a4e203766ffbea3146967ef25e9daf677b14dc6f6ed891
...
+[Polkadot website](https://www.polkadot.network/ecosystem/wallets/). See also the
+[Wallets](./wallets-index) section for more information about wallets and the
+[wallets and extensions](../general/wallets-and-extensions.md) page for wallets and browser
+extensions funded by the Polkadot/Kusama Treasuries or by the
+[Web3 Foundation Grants Program](../general/grants.md).
 
-More in-depth information about the above data can be found in the
-[substrate code base](https://github.com/paritytech/substrate/blob/2e7fde832b77b242269b136f1c3b6fffef86f9b6/frame/system/src/lib.rs#LL767C1-L781C24).
+Cold keys are generated on special devices such as those provided by [Ledger](../general/ledger.md).
+Additionally, you can generate your wallet using the [Polkadot Vault](../general/polkadot-vault.md)
```
Comment (profradha) [inline docs/learn/learn-accounts.md]:
```suggestion
Additionally, you can generate your account using the [Polkadot Vault](../general/polkadot-vault.md)
```

File: docs/learn/learn-accounts.md [hunk 6]
```diff
@@ -77,122 +68,54 @@ Secret seed (Private key): 0x056a6a4e203766ffbea3146967ef25e9daf677b14dc6f6ed891
...
 
-The `AccountData` structure defines the balance types in Substrate. The three types of balances
-include `free`, `reserved`, and `frozen`. The **usable** balance of the account is the amount that
-is `free` minus any funds considered `frozen`, while the **total** balance of the account is the sum
-of `free` and `reserved` funds. The `flags` describe extra information about the account.
+Usually, browser extensions and mobile devices have options to import accounts generated on cold
+devices. Note that the private keys of those accounts will remain on the cold device, meaning that
+you will always need the device to sign for any transaction. Exceptions exist where you can generate
+hot [proxy accounts](./learn-proxies.md) and sign on behalf of a cold account without using the cold
+device. This is practical, especially for routine transactions.
```
Comment (profradha) [inline docs/learn/learn-accounts.md]:
```suggestion
Usually, browser extensions and mobile devices have options to securely import accounts from cold
wallets. Note that the private keys of those accounts will remain on the cold wallet, meaning that
you will always need the device to sign any transaction. Exceptions exist where you can generate
hot wallet based [proxy accounts](./learn-proxies.md) and sign on behalf of a cold wallet account without connecting the cold device. This is practical, especially for transactions made frequently.
```

**Review by DrW3RK — COMMENTED** [review]
PR too big! Will keep reviewing it and add feedback over time.

**Comment by filippoweb3** [comment]
@DrW3RK I moved around most JS stuff to the Polkadot-JS guides. The Account page has changed, but is actually the only place where I actually changed the text a bit

**Comment by DrW3RK** [comment]
Ah, okay. Thanks for clarifying. I shall be done reviewing it soon then.

File: docs/learn/learn-accounts.md [hunk 7]
```diff
@@ -216,52 +137,49 @@ construct a valid transaction. Additional funds will need to be added to cover t
...
 
-:::
+In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} there are different types of
+balance depending on the account activity. Different balance types indicate whether your balance can
+be used for transfers, to pay fees, or must remain frozen and unused due to an on-chain requirement.
+Below is an example that displays different balance types of a Kusama account (note that the balance
+types are the same for a Polkadot account).
 
-## Resources
+![account_balance_types](../assets/account-balance-types.png)
```
Comment (profradha) [inline docs/learn/learn-accounts.md:157]: I think it is okay to mention that this screenshot is from Polkadot-JS UI. As this is not part of a tutorial the user may have to follow, it should be okay.

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**